backend/plugins/authorization.js

```diff
@@ -57,26 +57,32 @@ export function PlotlyOAuth(electron) {
         fetch(`${getSetting('PLOTLY_API_URL')}/v2/users/current`, {
             headers: {'Authorization': `Bearer ${plotlyAuthToken}`}
         })
-        .then(userRes => userRes.json().then(userMeta => {
+        .then(userRes => {
             if (userRes.status !== 200) {
-                res.json(401, {error: {message: 'Please login to access this page.'}});
-                return next(false);
+                return userRes.text().then(body => {
```

Comment:
Instead of checking for status code, we should check for `userRes.json().username` .. If it is `null`, it means that the user was not authenticated with Plotly (or the accessToken has expired)

Reply:
@tarzzz 
> Instead of checking for status code, we should check for userRes.json().username

Addressed in https://github.com/plotly/falcon-sql-client/pull/368/commits/bc901daf225297be8bd4198346b4124a0be87e41